### PR TITLE
changed version to 2.3.0

### PR DESCRIPTION
**Pull request by heejaechang**

### CODE REVIEW

**Comment by heejaechang** [comment]
@mavasani can you take a look?

**Comment by heejaechang** [comment]
thank you

File: build/Versions.props [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionBase>2.6.0</VersionBase>
+    <VersionBase>2.3.0</VersionBase>
```
Comment (mavasani) [inline build/Versions.props]: Prerelease label below should be beta2, we have already published beta1.

Comment (heejaechang) [inline build/Versions.props]: okay

**Comment by heejaechang** [comment]
retest this please

**Comment by heejaechang** [comment]
nuget restore issues.

**Comment by heejaechang** [comment]
@dotnet/roslyn-infrastructure nuget is keep failing 
https://ci.dot.net/job/dotnet_roslyn-analyzers/job/master/job/windows_debug_prtest/201/

**Comment by heejaechang** [comment]
retest prtest/windows/debug please

**Comment by tmat** [comment]
I guess we need to add the VBCSCOmpiler.exe killing script.

**Pull request merged**